releasenotes/notes/50804.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+issue:
+  - 50736
+releaseNotes:
+- |
+  **Added** Allow pods to be opted out of ambient capture using the `istio.io/dataplane-mode=none` label.
+  **Removed** the ability to opt-out pods from ambient capture using the `ambient.istio.io/redirection=disabled` annotation, as that is a status annotation reserved for the CNI.
```

Comment:
Didn't think we documented this anywhere, probably okay to not mention it.

Reply:
We haven't documented the old per-pod exclusion mechanism yet outside of code (IIRC, I will check), so I don't think we have to update docs for this and can just add net-new documentation for this.

Reply:
No I lied, we do mention it, I will open a docs PR.

Reply:
https://github.com/istio/istio.io/pull/15017/files

Reply:
Thanks @bleggett, SG!